src/components/Rating.tsx

```diff
@@ -0,0 +1,38 @@
+import starFilledIcon from '/icons/star_filled.svg';
+import starHalfIcon from '/icons/star_half.svg';
+import starHollowIcon from '/icons/star_hollow.svg';
+
+interface RecipeRatingProp {
+    rating: number;
+}
```

Comment:
I think this could be a `ProductRatingProp`.
```suggestion
interface ProductRatingProp {
    rating: number;
}
```

Reply:
Changed it.
https://github.com/dhruvdhaduk-simform/ecommerce/pull/2/commits/7da943a1a8f71e86ac9ee6406b42944c9e7a29b4